### PR TITLE
[tools] Fix wrong lost capitalization in translate.py and enabled af, lt translations

### PR DESCRIPTION
**Pull request by biodranik**

### CODE REVIEW

File: tools/python/translate.py [hunk 1]
```diff
@@ -48,7 +49,7 @@
     'it',
     'ja',
     'ko',
-#    'lt',
+    'lt',
```
Comment (vng) [inline tools/python/translate.py:52]: It was commented purposely, because categories don't support lt.

File: tools/python/translate.py [hunk 2]
```diff
@@ -15,6 +15,7 @@
 # Use DeepL when possible with a fall back to Google.
 # List of Google Translate target languages: https://cloud.google.com/translate/docs/languages
 GOOGLE_TARGET_LANGUAGES = [
+  'af',
```
Comment (vng) [inline tools/python/translate.py:18]: Same here, this translations will be invalid for categories.

**Review by vng — APPROVED** [review]
Why did you put all these changes into one commit? I see 2 or even 3 separate here.

**Comment by biodranik** [comment]
> Why did you put all these changes into one commit? I see 2 or even 3 separate here.

Good point, fixed.

**Pull request merged**